sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -113,6 +113,28 @@ export interface EntitiesResponse<T>
     Pick<PageSettings, "continuationToken">,
     Response {}
 
+/**
+ * Array of objects representing runtime info for multiple queues.
+ */
+export interface QueuesRuntimeInfoResponse extends Array<QueueProperties>, Response {}
+/**
+ * Represents result of create and update operations on queue.
+ */
+export interface QueueResponse extends QueueProperties, Response {}
+/**
+ * eTag is returned as part of the response for get requests and is sent as part of the update requests.
+ */
+export interface ETag {
+  /**
+   * eTag represents the resource state and an error is thrown for the update request if there is a
+   * mismatch in the eTag since that refers to the updated state of the resource.
+   */
+  eTag: string;
```

Comment:
We already chose to use `etag` in other libraries, like storage.

Reply:
Thanks, will update.